scripts/caput-pipeline

```diff
@@ -70,13 +70,32 @@ def queue(configfile, nosubmit=False):
         raise Exception('Job time is required.')
     clusterconf['time'] = conf['time']
 
+    if 'cluster' not in conf:
+        raise Exception('Cluster is required')
+    elif (conf['cluster'] == 'gpc'):
```

Comment:
Options should be `slurm` or `pbs` I would suggest. Where possible this repo should not hardcode CHIME specifics (though I think it's okay to set defaults that make *our* lives a little easier).